### PR TITLE
Added guide for auto updating node with docker

### PR DESCRIPTION
**Pull request by ArthurPatriot**
1. Guide for auto update node on docker with Watchtower.
2. Update deps.

### CODE REVIEW

File: docs/Community/phase-1/02-download-build-run-docker.md [hunk 1]
```diff
@@ -8,7 +8,8 @@
 - [Step 3: Edit `node.config`](#step-3-edit-nodeconfig)
   - [How to edit `node.config`?](#how-to-edit-nodeconfig)
 - [Step 4: Get the certificate](#step-4-get-the-certificate)
-- [And, finally, step 5: Start the node](#and-finally-step-5-start-the-node)
+- [Step 5: Start the node](#step-5-start-the-node)
+- [Step 6 (optional): Automate updates for node with Watchtower](#step-6-optional-automate-updates-for-node-with-watchtower)
```
Comment (EZabotkin) [inline docs/Community/phase-1/02-download-build-run-docker.md]: Step 6 (optional): Automated updates for node with Watchtower

File: docs/Community/phase-1/02-download-build-run-docker.md [hunk 2]
```diff
@@ -234,6 +235,21 @@ where:
 | `-p 41026:41026` <br/> `-p 1080:1080` <br/> `-p 1443:1443`                       | These commands specify all necessary local ports. In this examples ports `api`, `apis`, and `tpic` are used. You can specify any port in `node.config` file. `-p 41026:41026` is different for each chain |
 | `thepowerio/tpnode`                                                              | Path to Docker image.                                                                                                                                                                                     |
 
+## Step 6 (optional): Automate updates for node with Watchtower
+
+Every 24 hours, the [Watchtower](https://containrrr.dev/watchtower/) will check whether there is a new version of The Power Node and will update if so.
```
Comment (EZabotkin) [inline docs/Community/phase-1/02-download-build-run-docker.md]: Every 24 hours, the [Watchtower](https://containrrr.dev/watchtower/) will check if a new version of The Power Node is available and update it if so.

File: docs/Community/phase-1/02-download-build-run-docker.md [hunk 3]
```diff
@@ -234,6 +235,21 @@ where:
...
+
```
Comment (EZabotkin) [inline docs/Community/phase-1/02-download-build-run-docker.md:131]: Add a 3rd level subheader:

### Start a container

**Review by EZabotkin — CHANGES_REQUESTED** [review]
Need some minor changes

**Review by giDai7ja — APPROVED** [review]
@EZabotkin
Useful information
See that it fits well into the main documentation

**Review by giDai7ja — APPROVED** [review]

**Review by EZabotkin — APPROVED** [review]

**Pull request merged**